packages/monitoring/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Stackdriver Monitoring library for Node.js",
```

Comment:
To match our other descriptions, this would be "Stackdriver Monitoring Client Library for Node.js"